automation/submission-perf-inc.sh

```diff
@@ -143,7 +148,13 @@ launchSubmission() {
         optionalEntityNameField="\"entityName\":\"${entityName}\","
     fi
 
-    curl \
+    launchSubmissionResponse=$(curl \
+        -s \
+        -o /dev/null \
+        -w "%{http_code}" \
+        --connect-timeout 60 \
+        --retry 8 \
```

Comment:
Have we found we actually want 8 retries here? Likely isn't going to be a huge contributor to slowing what is already pretty slow CI, but seems high to me.